src/sections/wallet/addToken/modal/AddTokenListModal.tsx

```diff
@@ -20,6 +23,12 @@ import { AddTokenListSkeleton } from "sections/wallet/addToken/modal/skeleton/Ad
 import { useSettingsStore } from "state/store"
 import { theme } from "theme"
 
+export const UigcAssetId = createComponent({
```

Comment:
Why do you use that instead of AssetLogo?

Reply:
AssetLogo only works when user has the asset in LS, otherwise, symbol is unknown.

Its temporary solution until all logos are determined by internal ID and not symbol.